### DISCUSSION TITLE
Would you approve of adding a large number of locales?

### DISCUSSION THREAD

**Discussion by EgorK0rshun** [fluttercandies/flutter_wechat_assets_picker#485] [General]
Currently thinking about switching to your nice library, but mine is in quite a few languages. You have Chinese by default if there are no locales available. can I add about ten or so?

**Comment by AlexV525** [comment]
If you're willing to contribute localizations, make sure your fields match these requirements:
- Clear and precise about its functionality.
- Should not get updates frequently.
- Need semantics support (necessary if supported by TalkBack/VoiceOver) or fallback support to other locales that support semantics.
- Acts correctly in a screen record of all locale fields.
